tests/run_inputs.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+
+import sys
+import subprocess
+from subprocess import Popen, PIPE, STDOUT
+import os
+import re
+ 
+def main():
+    """This function finds input files, runs them, and prints a summary"""
+    flag = check_inputs()
+    input_path = "/Users/mbmcgarry/git/cycamore/../input"
+    cyclus_path =  "/Users/mbmcgarry/.local/include/cyclus/../../bin/cyclus"
+    files, catalogs, catalognames = get_files(input_path)
+    copy_catalogs(catalogs,cyclus_path.strip("cyclus"))
+    summ = Summary()
+    for name in files :
+        file_to_test = TestFile(cyclus_path, name, flag) 
+        file_to_test.run()
+        summ.add_to_summary(file_to_test)
+    clean_catalogs(cyclus_path.strip("cyclus"),catalognames)
+    summ.print_summary()
+
+def check_inputs():
+    """This function checks the input arguments"""
+    if len(sys.argv) > 2: 
+        print_usage()
+        sys.exit(1) 
+    elif len(sys.argv) == 2:
+        if re.search("-v*",sys.argv[1]):
+            flag = sys.argv[1]
+        else : 
+            print_usage()
+            sys.exit(1)
+    elif len(sys.argv) == 1 :
+        flag = "-v0"
+    return flag
+
+def print_usage() :
+    """This prints the proper way to treat the command line interface"""
+    print 'Usage: python run_inputs.py\n' + \
+            'Allowed Options : \n' + \
+            '-v arg                    output log verbosity. \n' + \
+            '                    \
+            Can be text: \n' + \
+            '                    \
+              LEV_ERROR (least verbose, default), LEV_WARN,\n' + \
+            '                    \
+              LEV_INFO1 (through 5), and LEV_DEBUG1 (through 5).\n' + \
+            '                    \
+            Or an integer:\n'+ \
+            '                    \
+              0 (LEV_ERROR equiv) through 11 (LEV_DEBUG5 equiv)\n'
+ 
+def get_files(path):
+    """This function walks the 'path' tree and finds input files"""
+    catalogs=[]
+    catalognames=[]
+    inputs=[]
+    for root, dirs, files in os.walk(path, followlinks=True):
+        if '.git' in dirs:
+            dirs.remove('.git')
+        for name in files: 
+            if re.search("\.xml",name) :
+                if re.search("recipebook",name) or \
+                        re.search("facilitycatalog",name):
+                    catalogs.append(os.path.join(root,name))
+                    catalognames.append(name)
+                else : 
+                    inputs.append(os.path.join(root, name))
+            else :
+                files.remove(name)
+    print "The catalogs to be moved are:"
```

Comment:
I'm confused about this tests/run_inputs.py.  This is not a file that I wrote, but it appears that it wasn't in the old EF - perhaps it is automatically generated and was supposed to be in .ignore?